mep/people/templates/people/borrowing_activities.html

```diff
@@ -60,31 +60,31 @@
     <p>No documented borrowing activity.</p>
     {% else %}
     <div class="wrapper">
-    <table id="activity-table">
+    <table id="activity-table" class="borrowing">
         <thead>
             <tr>
                 <th>Title</th>
                 <th>Author</th>
                 <th>Publication Date</th>
-                <th>Status</th>
+                <th>Type</th>
                 <th data-sort-default>Start Date</th>
                 <th>End Date</th>
             </tr>
         </thead>
         {% for event in event_list %}
         <tr>
-            <td class="title">{{ event.work.title }}</td>
-            <td class="author">{{ event.work.author_list }}</td>
-            <td class="pubdate">{{ event.work.year }}</td>
-            <td class="status">
-                {% if event.borrow %}{{ event.borrow.status }}
-                {% elif event.purchase %}Purchased{% endif %}
+            <td class="title">{{ event.work.title|default:'-' }}</td>
+            <td class="author">{{ event.work.author_list|default:'-' }}</td>
+            <td class="pubdate">{{ event.work.year|default:'-' }}</td>
+            <td class="type">
```

Comment:
I think we might want to use `with` for the event type, I'm not sure how efficient it is - I don't think it's cached. Also, it occurs to me that we should try prefetching borrows and purchases on the queryset - I think this should make it more efficient, and would probably be noticeable for borrowers with a lot of events (i.e. Killen).

Reply:
I was wondering that about a lot of these...would `event.work` get cached, and is it worth it binding that in a `with` also? Love the idea of prefetching.

Reply:
Oh, we should prefetch works too.  I think once they're prefetched the caching may not be a problem? I could try to take a look at the prefetching.

Reply:
either way - if you wanna push to this branch it should be fine since that should be happening in the view and I'm not working there, but I could take a look at it also